### ISSUE TITLE
Passwords of insufficient length do not trigger screen reader announcements

### ISSUE THREAD

**Issue by KrolPolski** [Welhox/Transcendence#82]
![Image](https://github.com/user-attachments/assets/dbc2eb87-3614-4a80-8215-706562e14a96)

When the passwords are not long enough on the register page, we get a small alert from somewhere that doesn't trigger any screen reader announcements and doesn't use our error message <p> element either. The image above is from Chrome but similar behavior occurs in Firefox.

I wonder if this is coming from the browser default behavior.

**Comment by KrolPolski** [comment]
This has been fixed in the registration-and-accessibility branch.
